stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/message/composer/MessageComposerViewModelBinding.kt

```diff
@@ -0,0 +1,24 @@
+package io.getstream.chat.android.ui.message.composer
+
+import androidx.lifecycle.LifecycleOwner
+import androidx.lifecycle.coroutineScope
+import kotlinx.coroutines.flow.collect
+import kotlinx.coroutines.launch
+
+public fun MessageComposerViewModel.bindView(view: MessageComposerView, lifecycleOwner: LifecycleOwner) {
+
+    view.onSendMessageAction = {
```

Comment:
So basically if the user decides to bind the view to the ViewModel, we override the `onSendMessageAction`? As well as `onInputTextChanged`. 

How do our users customize these then? If they want to intercept the changes and all.

Do they first have to bind the view and then override the changes?

Reply:
I've simplified this in be6f49d. Letting the custom/default content elements communicate with view model directly. This will simplify actions flow a lot.

Reply:
I think it's not a matter of simplification - we just need to expose the ability for our users to override the UI actions and intercept them - it's what we've been exposing for a while. 

The issue here was that the user can set those actions before calling `bindView()` and then call bind and those actions would be overriden.

We can build the logic in a way that we allow for the callbacks to be overriden and then we can set up default behavior for users so they don't have to do anything by default.